conf/test.config

```diff
@@ -253,29 +253,32 @@ process {
     }
 }
 
-// ENABLE CI containers for testing
+// Enable container engines/virtualisation envs for CI testing
+// only works when specified with the profile ENV
+// otherwise tests can be done with the regular provided profiles
 if (System.getenv('PROFILE')) {
     if ("$PROFILE" == "conda") {
-            params.enable_conda    = true
-            docker.enabled         = false
-            singularity.enabled    = false
-            podman.enabled         = false
-            shifter.enabled        = false
-            charliecloud.enabled   = false
+        conda.createTimeout                      = "120 min"
+        params.enable_conda                      = true
+        charliecloud.enabled                     = false
+        docker.enabled                           = false
+        podman.enabled                           = false
+        shifter.enabled                          = false
+        singularity.enabled                      = false
     } else if ("$PROFILE" == "docker") {
-            docker.enabled         = true
-            docker.userEmulation   = { params.use_gatk_spark ? false : true }.call()
-            singularity.enabled    = false
-            podman.enabled         = false
-            shifter.enabled        = false
-            charliecloud.enabled   = false
-            fixOwnership = true
```

Comment:
Decided to remove that instead of fixing it, as it was causing the strelkabp test to fail...

Reply:
only now?

Reply:
ok